ui/app/src/components/PreviewBrowseComponentsPanel/PreviewBrowseComponentsPanel.tsx

```diff
@@ -66,7 +66,7 @@ export function PreviewBrowseComponentsPanel() {
 
   useEffect(() => {
     if (siteId && contentTypesBranch.isFetching === false) {
-      dispatch(fetchComponentsByContentType({}));
+      dispatch(fetchComponentsByContentType({ sortBy: 'internalName', sortOrder: 'asc' }));
```

Comment:
I don't see the types for the `fetchComponentsByContentType` action receiving sorting related stuff. Not sure if the service does but there seems to be a missing piece. 

Also, if the service does, what's the need of the sort above on line 55?

Reply:
Yeah, updated `ComponentsContentTypeParams`. Also, the [reducer](https://github.com/craftercms/studio-ui/blob/support/4.1.x/ui/app/src/state/reducers/preview.ts#L481) sets the query that is used for the search.

The sort on line 55 is for the content-types, the sort params are for the items of the content-type